ansible/roles/common/tasks/tools.yml

```diff
@@ -46,11 +46,17 @@
 
 - name: create scheduling cron job
   cron:
-    name: schedule passes
+    name: schedule passes dayly
```

Comment:
Can you update this to be "daily" for me?

Reply:
Ups :)